core/prop.h

```diff
@@ -27,18 +27,11 @@ class Property
   Property(const TransitionSystem & ts, const smt::Term & p,
            std::string name="");
 
-  /** Copies property to a new solver
-   *  @param prop the property to copy
-   *  @param tt the term translator to use
-   *  @return a property using the solver in tt
-   */
-  Property(Property & prop, smt::TermTranslator & tt);
-
   ~Property();
 
   const smt::Term prop() const { return prop_; }
 
-  TransitionSystem & transition_system() { return ts_; }
+  const TransitionSystem & transition_system() const { return ts_; }
```

Comment:
Should we still include the transition system at all? I was thinking it could just be the solver, name, and the property term.

Reply:
I see -- it's in the second part of this PR.